### PR TITLE
Use link.hypha.coop conventions in template

### PR DESCRIPTION
**Pull request by patcon**
Just wanted to demonstrate how commons links can start becoming easier to find. No rush to merge this until we have the infra convo and an official proposal

Also, bonus: created a little screencast to show how easy navigating them becomes when you use custom browser search engines :)

screenshot of adding/using shortlinks as keywords with custom search engine set in browser

### CODE REVIEW

File: -meeting-template.md [hunk 1]
```diff
@@ -48,10 +48,10 @@ Notetaker: ...
...
-[gdrive]: https://drive.google.com/drive/u/0/folders/14KYnYwOEK3InYZ3jCn-Gtf5q430sE9oc
-[biz-wg]: https://loomio.hypha.coop/g/ojZI2bPl/working-groups-business-planning
-[fin-wg]: https://loomio.hypha.coop/g/sRPwaorg/working-groups-finance
-[gov-wg]: https://loomio.hypha.coop/g/BaAj6dQn/working-groups-governance-by-laws-incorporation-articles-gm-
-[ino-wg]: https://loomio.hypha.coop/g/KvARWad7/working-groups-infrastructure-and-operations
+[gdrive]: https://link.hypha.coop/gdrive
+[biz-wg]: https://link.hypha.coop/biz-wg
+[fin-wg]: https://link.hypha.coop/fin-wg
+[gov-wg]: https://link.hypha.coop/gov-wg
+[ops-wg]: https://link.hypha.coop/ops-wg
```
Comment (benhylau) [inline -meeting-template.md]: I am pro-shortlink and prefer the shortlinks mapping is on a plaintext file in github like a yaml:

```
gdrive: https://drive.google.com/drive/u/0/folders/14KYnYwOEK3InYZ3jCn-Gtf5q430sE9oc
biz-wg: loomio.hypha.coop/g/ojZI2bPl/working-groups-business-planning
fin-wg: https://loomio.hypha.coop/g/sRPwaorg/working-groups-finance
gov-wg: https://loomio.hypha.coop/g/BaAj6dQn/working-groups-governance-by-laws-incorporation-articles-gm-
ops-wg: https://loomio.hypha.coop/g/KvARWad7/working-groups-infrastructure-and-operations
```

@dcwalk @ASoTNetworks I like to hear your thoughts on this.

Comment (patcon) [reply -meeting-template.md]: Not a hard sell on this, but the rationale was this: we learn that they exist through seeing them on link hover, not going out of the way to browse the spreadsheet :)

Will defer to collective on this though!

Comment (dcwalk) [reply -meeting-template.md]: I'm fine on this!

**Comment by dcwalk** [comment]
Can your explorable definition of using custom domain search,  links make it to the handbook @patcon?

**Comment by dcwalk** [comment]
@patcon this has been +1'd by folks and discussed at the infra meeting. I'd say it is ready to merge in, do you want to do the honours?

**Pull request merged**